content/user/0ee1bcb8-f26c-4809-8f58-3efb932dbe35.json

```diff
@@ -0,0 +1,103 @@
+{
+    "_links": {
+        "self": {
+            "href": "http:\/\/default\/user\/5?_format=hal_json"
+        },
+        "type": {
+            "href": "http:\/\/drupal.org\/rest\/type\/user\/user"
+        }
+    },
+    "uid": [
+        {
+            "value": 5
+        }
+    ],
+    "uuid": [
+        {
+            "value": "0ee1bcb8-f26c-4809-8f58-3efb932dbe35"
```

Comment:
add the new user UUID's to the stanford_profile.info.yml file for default content tracking